code/modules/hydroponics/grown/nymph.dm

```diff
@@ -11,6 +11,44 @@
 	production = 1
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
+	var/next_ping_at = 0
+
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/attack_ghost(mob/dead/observer/O, obj/machinery/hydroponics/H)
+	if(!planted && !H)
+		if(isobserver(O) && (world.time >= next_ping_at))
+			next_ping_at = world.time + (20 SECONDS)
+			visible_message("<span class='notice'>[src] rustles gently, as if moved by a gentle breeze.</span>")
+		return
+	if(!H.harvest)
+		to_chat(O, "<span class='warning'>[src] is not yet ready.</span>")
+		return
+	if(H.dead)
+		to_chat(O, "<span class='warning'>[src] is dead!</span>")
+		return
+	if(!(O in GLOB.respawnable_list))
+		to_chat(O, "<span class='warning'>You are not permitted to rejoin the round.</span>")
+		return
+	else if(cannotPossess(O))
+		to_chat(O, "<span class='warning'>You have enabled antag HUD and are unable to re-enter the round.</span>")
+		return
+	var/nymph_ask = alert("Become a Diona Nymph? You will not be able to be cloned!", "Diona Nymph Pod", "Yes", "No")
+	if(nymph_ask == "No" || !src || QDELETED(src))
```

Comment:
`QDELETED` already catches `!src`. `!src` also should not happen since the proc will just stop then